src/puppet-web/firer.ts

```diff
@@ -367,7 +366,7 @@ async function checkRoomTopic(m: Message): Promise<void> {
   if (/^You$/.test(changer) || /^你$/.test(changer)) {
     changerContact = Contact.load(this.userId)
   } else {
-    changerContact = room.member(changer)
+    changerContact = room.member({remark: changer}) || room.member({nick: changer})
```

Comment:
> We can keep room.member(inviteeList[i]) here, after you follow the review for member() parameters suggestion.

I don't agree with this suggestion, because of the following reason:

We keep function `room.member()` work, but it should work only with the function find member by nickName.    
We cannot use it both find remark and nickName, because one can set a remark same as nick, then it would be confused. 

So, what do you think about it?